packages/dhis2/test/index.test.js

```diff
@@ -149,22 +149,43 @@ describe('get', () => {
   });
 });
 
-describe('helperfunctions', () => {
+describe.only('helperfunctions', () => {
   it('should use the new tracker for enrollments', () => {
     const result = shouldUseNewTracker('enrollments');
     expect(result).to.be.true;
   });
 
   it('should use the new tracker for events', () => {
+    const result = shouldUseNewTracker('events');
+    expect(result).to.be.true;
+  });
+
+  it('should use the new tracker for relationships', () => {
+    const result = shouldUseNewTracker('relationships');
+    expect(result).to.be.true;
+  });
+
+  it('should use the new tracker for new tracker events', () => {
     const result = shouldUseNewTracker('tracker/events');
     expect(result).to.be.true;
   });
 
+  it('should use the new tracker for trackedEntityInstance', () => {
+    const result = shouldUseNewTracker('trackedEntityInstance');
+    expect(result).to.be.true;
+  });
+
   it('should use the new tracker for trackedEntities', () => {
     const result = shouldUseNewTracker('trackedEntities');
     expect(result).to.be.true;
   });
 
+  it('should use the new tracker for new tracker generally', () => {
```

Comment:
Hmm. Question mark on this actually.

`create` should take a resourceType - not a path. So `create('enrollments')`, not `create('tracker/enrollments')`. Because create has opinions and does stuff for you.

If you want a custom create with the new tracker, do `post('tracker/enrollments')`